Runar/Runar/Services/Persistence layer/APIClient.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  APIClient.swift
+//  Runar
+//
+//  Created by Andrei on 11.02.2023.
+//
+
+import Foundation
+
+// TODO: - Should be delete after testing
+enum APIClientError: Error {
+    case invalidResponse
+}
+
+final class APIClient<T: Codable> {
+    func fetch(from url: URL) async throws -> T {
+        let (data, response) = try await URLSession.shared.data(from: url)
+        
+        guard
+            let response = response as? HTTPURLResponse,
+            response.statusCode == 200
```

Comment:
Create, please, separated extension for response codes and name it, for example, successCode.

Reply:
Can you clerify please, what is the main purpose of this extension, especially if we use APIClient only for test v2 API